widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts

```diff
@@ -3,5 +3,8 @@ import { styled } from '@rango-dev/ui';
 export const Blockchains = styled('div', {
   display: 'grid',
   gap: '$10',
-  gridTemplateColumns: 'repeat(6, minmax(0, 1fr))',
+  gridTemplateColumns: 'repeat(4, minmax(0, 1fr))',
```

Comment:
Now blockchain chips are not equal in size. You can solve this problem by setting `width: '100%'` for BlockchainChip.

<img width="223" alt="image" src="https://github.com/rango-exchange/rango-client/assets/154742563/4ad00bb0-512d-4e3b-a43d-3a951631421d">